### ISSUE TITLE
Directly measuring compression information loss on my sequences

### ISSUE THREAD

**Issue by liorglick** [amyxlu/cheap-proteins#6]
I'd like to run some of the pretrained models on my sequences and assess the information loss using different shortening factors and channel numbers. In the paper, you achieve this through measuring the performance of predictors trained using the compressed embeddings. What I am looking for is a more direct measure - I want to compare the original ESM embeddings to the ones extracted from the last layer of the hourglass model.
I was able to get the later using `pipeline.decode(emb, mask)`, which returned matrices of size 327x16 in the case of the usage example notebook.
How can I get the original ESM embeddings (with dimensions Lx1024)?

**Comment by liorglick** [comment]
OK, I might have figured it out:
```
res = pipeline.esmfold_embed_only_module.infer_embedding(sequences)
res['s']
```
returns the original ESM embeddings. Is that correct?

**Comment by amyxlu** [comment]
yup! that's correct :)
